AndelaOSP/art-dashboard
# #159816854 Add modals' loading icons

## What does this PR do?

- adds loading icons to asset Makes, Types, Model Number, Sub categories
   and on  add asset modals.

## Description of Task to be completed?
**Steps to reproduce**

- Login to app
- Click on the Add asset Button
- On the displayed modal, click on any of the dropdowns.
- Due to delay in fetching data, you will see No results Found

**Expected**

- User sees a loading icon on modal as data is being fetched for the dropdowns

**Actual**

- No results Found is displayed as the results are being fetched and after some seconds the message is replaced with loaded data

## How should this be manually tested?

- Once logged in, click on the `Add Asset` button, as the dropdown details are being fetched, you will see the loading icon on the modal.

- Go to the asset lists page. Click any of the + icons, while the data for the dropdown on the modal is being fetched, you will see a loading icon.

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/159816854

## Any background context you want to add?
N/a
## Important notes
N/a
## Packages installed
N/a
## Deployment note
N/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="503" alt="screen shot 2018-08-23 at 17 06 01" src="https://user-images.githubusercontent.com/27014080/44530383-e5100c80-a6f6-11e8-920d-ae645146dacf.png">

<img width="514" alt="screen shot 2018-08-23 at 17 06 52" src="https://user-images.githubusercontent.com/27014080/44530417-f822dc80-a6f6-11e8-86bc-a023f2e02da3.png">
